tests/platform/cloud/AcceptanceTests_untrustedSettingsGet.txt

```diff
@@ -1,7 +1,7 @@
 AcceptanceTests_untrustedSettingsGet.js
 
 Description:
-This is a series of tests using the online flowmanager to test the workflow that GPII app installations request filtered user settings for onboarded solutions that are authorized by users to access some or all of their preferences. Before requesting untrusted settings, the GPII app installation must request an access tokens from GPII Cloud using OAuth2 resource owner GPII token grant. This access token is then sent along with the untrusted setting request to represent the authorization of the GPII app installation to user settings.
+This is a series of tests using the online flowmanager to test the workflow that GPII app installations request settings in the ontology of preferences. Before requesting untrusted settings, the GPII app installation must request an access tokens from GPII Cloud using OAuth2 resource owner GPII token grant. This access token is then sent along with the untrusted setting request to represent the authorization of the GPII app installation to user settings.
```

Comment:
Minor text note: should be singular "token" in "Before requesting untrusted settings, the GPII app installation must request an access **tokens** from GPII...".